modules/mongodb/manifests/s3backup/package.pp

```diff
@@ -1,9 +1,9 @@
 #
 class mongodb::s3backup::package {
 
-  $packages = ['boto3', 'python-dateutil', 'simplejson']
+  
 
-  package { $packages:
+  package { 's3cmd' :
```

Comment:
Yes.  The .s3cfg is used to store things like Access/Secret keys as well as gpg keys and sometimes transport config.  We are using the default https while the module i.e. envdir...etc  handles things like Access\Secret keys and gpg fingerprints.